src/main/asciidoc/transactions.adoc

```diff
@@ -33,82 +41,257 @@ static class Config extends AbstractCouchbaseConfiguration {
-  transactions.run(ctx -> {
+.Transaction Access Using TransactionalOperator.execute()
+====
+[source,java]
+----
+@Autowired TransactionalOperator txOperator;
 
-   Airline airline = new Airline("demo-airline", "at");
-   CouchbaseDocument target = new CouchbaseDocument();
-   mappingCouchbaseConverter.write(airline, target);
+    Person person = rxCBTmpl.insertById(Person.class).inCollection(cName).one(WalterWhite).block();
+    Flux<Person> result = txOperator.execute((ctx) -> rxCBTmpl.findById(Person.class).one(person.id())
+        .flatMap(p -> rxCBTmpl.replaceById(Person.class).one(p.withFirstName("Walt"))));
+    result.blockLast();
+----
+====
 
-   ctx.insert(couchbaseClientFactory.getDefaultCollection(), target.getId(), target.getContent());
+.Transaction Access Using transactional()
+====
+[source,java]
+----
+@Autowired TransactionalOperator txOperator;
 
-   ctx.commit();
-  });
-}
+    Person person = rxCBTmpl.insertById(Person.class).inCollection(cName).one(WalterWhite).block();
+    Mono<Person> result = rxCBTmpl.findById(Person.class).one(person.id())
+        .flatMap(p -> rxCBTmpl.replaceById(Person.class).one(p.withFirstName("Walt")))
+        .as(txOperator::transactional);
+    result.block();
 ----
 ====
+.Transaction Access  simulating @Transactional
+====
+[source,java]
+----
+@Service
+@Component
+class PersonService {
+
+    final ReactiveCouchbaseOperations personOperationsRx;
+    final TransactionalOperator transactionalOperator;
+
+    public PersonService(ReactiveCouchbaseOperations opsRx, TransactionalOperator transactionalOperator) {
+	this.personOperationsRx = opsRx;
+        this.transactionalOperator = transactionalOperator;
+    }
 
-The same approach can be used on read:
+    // this method does not have the @Transactional annotation, 
+    // but the as(trasnactionalOperator::transactional)
+    // makes it behave as though it does.
+    public Mono<Person> declarativeFindReplacePersonReactive(Person person) {
+        return personOperationsRx.findById(Person.class).one(person.id())
+                .flatMap(p -> personOperationsRx.replaceById(Person.class).one(p.withFirstName(person.getFirstname())))
+                .as(transactionalOperator::transactional);
+    }
+----
+====
 
-.Transaction Conversion on Read
+== Transactions with Only the SDK
+.Transaction Access - Blocking
 ====
 [source,java]
 ----
-TransactionGetResult getResult = ctx.get(couchbaseClientFactory.getDefaultCollection(), "doc-id");
+@Autowired CouchbaseTemplate template;
 
-CouchbaseDocument source = new CouchbaseDocument(getResult.id());
-source.setContent(getResult.contentAsObject());
-Airline read = mappingCouchbaseConverter.read(Airline.class, source);
+    TransactionResult result = template.getCouchbaseClientFactory().getCluster().transactions().
+            run(ctx -> 
+                { 
+                    return template.insertById(Person.class).one(WalterWhite); 
+                },
+                options);
```

Comment:
`options` isn't in this example - in general I think it's best for examples to compile on a C&P